Makefile

```diff
@@ -1,23 +1,128 @@
+ONNX_PYFILES := $(shell find examples/post_training_quantization/onnx -type f -name "*.py")
 pylint-onnx:
 	pylint --rcfile .pylintrc               \
 		nncf/experimental/onnx              \
 		nncf/quantization                   \
 		tests/onnx                          \
-		$(PYFILES)                         
+		$(ONNX_PYFILES)
 
 test-install-onnx:
-	pytest tests/cross_fw/install/ --backend onnx
+	pytest tests/cross_fw/install/ -s       \
+		--backend onnx                      \
+		--junitxml ${JUNITXML_PATH}
+
+###############################################################################
+# OpenVino backend
+install-openvino-test:
+	pip install -U pip
+	pip install -e .[openvino]
+	pip install -r tests/openvino/requirements.txt
+	pip install -r tests/cross_fw/install/requirements.txt
+	pip install -r examples/experimental/openvino/bert/requirements.txt
+	pip install -r examples/experimental/openvino/yolo_v5/requirements.txt
+
+install-openvino-dev: install-openvino-test
+	pip install pylint==$(PYLINT_VERSION)
+
+test-openvino:
+	pytest tests/openvino --junitxml ${JUNITXML_PATH}
+
+pylint-openvino:
+	pylint --rcfile .pylintrc               \
+		nncf/openvino/                      \
+		tests/openvino/                     \
+		examples/experimental/openvino/
+
+test-install-openvino:
+	pytest tests/cross_fw/install -s        \
+		--backend openvino                  \
+		--junitxml ${JUNITXML_PATH}
+
+###############################################################################
+# TensorFlow backend
+install-tensorflow-test:
+	pip install -U pip
+	pip install -e .[tf]
+	pip install -r tests/tensorflow/requirements.txt
+	pip install -r tests/cross_fw/install/requirements.txt
+	pip install -r examples/tensorflow/requirements.txt
+
+install-tensorflow-dev: install-tensorflow-test
+	pip install pylint==$(PYLINT_VERSION)
+
+test-tensorflow:
+	pytest tests/common tests/tensorflow    \
+		--junitxml ${JUNITXML_PATH}         \
+		$(DATA_ARG)
+
+pylint-tensorflow:
+	pylint --rcfile .pylintrc               \
+		nncf/tensorflow                     \
+		nncf/experimental/tensorflow        \
+		tests/tensorflow                    \
+		tests/experimental/tensorflow       \
+		examples/tensorflow
+
+test-install-tensorflow:
+	pytest tests/cross_fw/install/ -s --backend tf --junitxml ${JUNITXML_PATH}
+
+###############################################################################
+# PyTorch backend
+install-torch-test:
+	pip install -U pip
+	pip install -e .[torch]
+	pip install -r tests/torch/requirements.txt
+	pip install -r tests/cross_fw/install/requirements.txt
+	pip install -r examples/torch/requirements.txt
+
+install-torch-dev: install-torch-test
+	pip install pylint==$(PYLINT_VERSION)
+
+test-torch:
+	pytest tests/common tests/torch --junitxml ${JUNITXML_PATH} $(DATA_ARG)
+
+pylint-torch:
+	pylint --rcfile .pylintrc               \
+		nncf/common                         \
```

Comment:
Have you checked the output of the pylint in that way?
It's a bit different from what we have in CI now, there might be some additional pylint warnings.

Reply:
Yes, I checked locally and everything is fine.